app/scripts/modules/amazon/src/serverGroup/details/serverGroupDetails.aws.controller.js

```diff
@@ -154,6 +154,11 @@ module.exports = angular.module('spinnaker.amazon.serverGroup.details.controller
                 this.changeConfig = {
                   metadata: get(this.serverGroup.entityTags, 'creationMetadata')
                 };
+                if (this.serverGroup.buildInfo.jenkins) {
```

Comment:
The `changeConfig` is just a means of getting data into the changes link component.  Which is why I want to normalize the input.  Appreciate the additional info on jenkins, that's helpful.